integration/dynamodb_test.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/containous/traefik/integration/utils"
+	"github.com/containous/traefik/types"
+	"github.com/go-check/check"
+	checker "github.com/vdemeester/shakers"
+)
+
+type DynamoDBSuite struct {
+	BaseSuite
+}
+
+type DynamoDBBackendItem struct {
+	ID      string        `dynamodbav:"id"`
+	Name    string        `dynamodbav:"name"`
+	Backend types.Backend `dynamodbav:"backend"`
+}
+
+type DynamoDBFrontendItem struct {
+	ID       string         `dynamodbav:"id"`
+	Name     string         `dynamodbav:"name"`
+	Frontend types.Frontend `dynamodbav:"frontend"`
+}
```

Comment:
Yeah I thought about that but I'm not sure how that works. In dynamo the attributes need to be named frontend or backend. If I leave the top level struct tag empty does it inherit the tag or something like that?